hepcrawl/extractors/oup_parser.py

```diff
@@ -71,7 +71,8 @@ def parse_node(self, response, node):
         record.add_xpath('journal_artid', '//elocation-id/text()')
 
         published_date = self._get_published_date(node)
-        record.add_value('journal_year', int(published_date[:4]))
+        volume = self.get_volume_year(node)
+        record.add_value('journal_year', int(volume))
```

Comment:
Are you sure the published date for OUP is always just a year? can it be ``12-2022`` or `01-12-2022`?

Reply:
I couldn't find different. It's just a year put in <volume> tags, but I cannot be 100% sure is it different, it might be, if they will decide to change. But for now, the articles that I checked had just a year.
In the future, we should as XML schemas from publishers